content/docs/tutorials/smartmodule-development.md

```diff
@@ -176,7 +199,30 @@ Consuming records from the beginning of topic 'filter-test'
 42
 ```
 
-You now know  the development workflow for SmartModules with `smdk`. You can now generate your own project and process your own data.
+## Publish to SmartModule Hub
+
+As bonus, let's [publish] this SmartModule to [SmartMoudle Hub].
+
+%copy first-line%
+```bash
+$ smdk publish
+Creating package my-group/regex-filter@0.1.0
+.. fill out info in hub/package-meta.yaml
+Package hub/regex-filter-0.1.0.ipkg created
+Package uploaded!
+```
+
+Let's double check that the SmartModule is available for download:
+
+%copy first-line%
+```bash
+$ fluvio hub list
+  SMARTMODULE                    
+  my-group/regex-filter@0.1.0
+```
+
+
+You now know  the development workflow for SmartModules with `smdk` and publish to `the HUb`. You can now generate your own project that processes data. At last, you can shared the SmartModule with your team.
```

Comment:
"You now know  the" -- there are 2 spaces between "know" and "the"

`the HUb` -- Casing typo. Plus this is an unconventional use of the code markup. To most readers, it isn't clear that it isn't a command to run in contrast to the actual command `smdk`.